src/app/shared/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class AppConstants {
```

Comment:
Are we using this in multiple places? If not make it local variable of that component